perllib/FixMyStreet/App/Controller/Admin/ResponsePriorities.pm

```diff
@@ -0,0 +1,125 @@
+package FixMyStreet::App::Controller::Admin::ResponsePriorities;
+use Moose;
+use namespace::autoclean;
+
+BEGIN { extends 'Catalyst::Controller'; }
+
+use Path::Class;
+use POSIX qw(strftime strcoll);
+use Digest::SHA qw(sha1_hex);
+use mySociety::EmailUtil qw(is_valid_email);
+use DateTime::Format::Strptime;
+use List::Util 'first';
+
+
+sub begin : Private {
+    my ( $self, $c ) = @_;
+
+    $c->forward('/admin/begin');
+}
+
+sub auto : Private {
+    my ( $self, $c ) = @_;
+
+    $c->forward('/admin/auto');
+}
```

Comment:
All `auto` functions in the chain are called (unlike `begin` or `end`) so this function shouldn't be needed, if it is, something must be going wrong somewhere :)